### ISSUE TITLE
Configuring Remote Syslog in New Interface Does Not Restart Service

### ISSUE THREAD

**Issue by TotalGriffLock** [opnsense/core#3740]
If you configure a new remote syslog destination in the new "Logging / targets" UI, when you click apply the appropriate config is written to syslog-ng, but syslog-ng is not restarted or sent a SIGHUP to re-read the config file. A manual restart of syslog-ng is required to apply the changes.

Steps to recreate:
Configure remote syslog destination in Logging / targets
Apply changes
Monitor remote syslog destination, note that messages do not appear
Restart syslog-ng service on opnsense
Monitor remote syslog destination, note that messages do appear

Tested on OPNsense 19.7.4_1-amd64

**Comment by AdSchellevis** [comment]
On my end it reloads its configuration, below the output after apply
```
clog -f /var/log/system.log
Oct  1 18:31:24 OPNsense syslog-ng[47184]: Configuration reload request received, reloading configuration; 
Oct  1 18:31:24 OPNsense syslog-ng[47184]: Configuration reload finished; 
```

**Comment by TotalGriffLock** [comment]
Realised I'm testing on the wrong firewall and this one isn't fully up to date. It doesn't work in OPNsense 19.7-amd64. My mistake - I'll update it and see if the problem persists.

**Comment by TotalGriffLock** [comment]
It appears this was fixed in 19.7.1. Closing this report. Sorry!
